### PR TITLE
cloudformation integration tests

### PR DESCRIPTION
**Pull request by prasadkatti**
##### SUMMARY
Integration testsuite for cloudformation and cloudformation_info

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
cloudformation
cloudformation_info

##### ADDITIONAL INFORMATION
This testsuite is not comprehensive, but it is a start. I have added some comments describing what parts are tested and what still needs to be added later.

CI update is needed to make this work. PR - https://github.com/mattclay/aws-terminator/pull/82

### CODE REVIEW

File: test/integration/targets/cloudformation/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+        state: absent
+      check_mode: yes
+      register: cf_stack
+
+    - name: check task return attributes
+      assert:
+        that:
+          - not cf_stack.changed
+          - "'msg' in cf_stack"
+          - >-
```
Comment (jillr) [inline test/integration/targets/cloudformation/tasks/main.yml:318]:
```suggestion
          -
```
I don't believe this needs to be multiline

**Review by jillr — COMMENTED** [review]
One small nit, otherwise looks good.

File: test/integration/targets/cloudformation/tasks/main.yml (hunk 1 above)
Comment (prasadkatti) [reply test/integration/targets/cloudformation/tasks/main.yml:318]: I believe it does. The quote in `doesn't` kind of necessitates it. I asked about it on IRC and @tremble suggested the multiline trick.

Comment (jillr) [reply test/integration/targets/cloudformation/tasks/main.yml:318]: Oh apologies, I completely missed the apostrophe in the message - you're correct.

**Review by jillr — APPROVED** [review]

**Pull request merged**